### PR TITLE
Feature/save and load

### PR DESCRIPTION
**Pull request by lennczar**

### CODE REVIEW

**Comment by lennczar** [comment]
For now, just JSON works, and it will also be stored as pure JSON, not base64 encoded like on the proposals. I think this is already a valuable contribution so i opened the PR for review. If this is merged, please don't delete the branch, I will continue to make improvements and add features later down the road on this branch.

**Comment by mohamedalichelbi** [comment]
### TODO
- [ ]  make JSON use Base64

File: src/families/near/transfer.js [hunk 1]
```diff
@@ -69,6 +69,18 @@ export default class Transfer extends BaseTask {
...
+
+            for (let e in this.baseErrors)
+                this.errors[e].validOrNull(this.call[e])
+
+            this.errors.receiver.validOrNull(this.call.args.value.receiver_id);
+            this.errors.amount.validOrNull(this.call.args.value.amount);
+
+            WALLET.then(() => this.updateFT());
+
+        }).bind(this)
```
Comment (mohamedalichelbi) [inline src/families/near/transfer.js:83]: Do we still need this? I think it's from some old commit for fixing cloning bugs

File: src/families/base.js [hunk 2]
```diff
@@ -11,6 +11,7 @@ export default class BaseTask extends Component {
 
     uniqueClassName = "base-task";
     call;
+    loadErrors;
```
Comment (mohamedalichelbi) [inline src/families/base.js:14]: Might be from an old fix of cloning bugs? can you confirm if we still need it or not?

**Review by mohamedalichelbi — COMMENTED** [review]
Looks great! good job

**Pull request merged**